app/models/scheduled_database_check_job.rb

```diff
@@ -0,0 +1,18 @@
+class ScheduledDatabaseCheckJob
+  def self.perform(job_type)
+    case job_type
+    when 'clean_subscription_series'
+      Subscription.where(subscribable_type: "Series").includes(:subscribable).find_each do |s| 
+        s.delete if s.name.nil? && Series.find_by_id(s.subscribable_id).nil?
```

Comment:
Have we come across any instances where the name was nil _but_ there was a series with the matching id? In cases like that, should we try to repair the subscription somehow, or maybe just delete it anyway? The name being nil is what causes the 500 error, so leaving the subscription in its present state doesn't seem like a good idea.